lib/membrane/video_compositor/scene/object/input/static_frame.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Membrane.VideoCompositor.Scene.Object.Input.StaticFrame do
```

Comment:
I'm also not 100% sure about having `Input` in the module path (or whatever that's called in elixir :roll_eyes:) and creating a separate directory just for `StaticFrame` and `Video`. It seems to put more cognitive load on the person reading the code with not much benefit (since we're only moving two files to the separate folder)

Reply:
@DominikWolek What do you think?

Reply:
It's called module name. I agree, since we don't have `scene/object/input.ex` file

I'd also flatten other parts of the structure, e.g. `.Scene.Object`to `.Object`

Reply:
I changed the project module's structure. It's quite a major change, so can you check it out and let me know whether you like it? @DominikWolek @jerzywilczek